src/ecmascript/spidermonkey/console.h

```diff
@@ -0,0 +1,11 @@
+
+#ifndef EL__ECMASCRIPT_SPIDERMONKEY_CONSOLE_H
+#define EL__ECMASCRIPT_SPIDERMONKEY_CONSOLE_H
+
+#include "ecmascript/spidermonkey/util.h"
+
+extern const JSClass console_class;
+extern const spidermonkeyFunctionSpec console_funcs[];
+extern const JSPropertySpec console_props[];
+
+#endif
```

Comment:
This code must be compilable. Now it is not, I'm afraid.
At least with autotools, but better with both autotools and meson.

Reply:
Please prepare, compilable version.

Reply:
What does that log? I just did compile it. I don't have meson thou.

Reply:
How should I resolve this? I do this for compile the spidermonkey:
1) ./autgen.sh
2) ./configure --with-spidermonkey
3) edit the Makefile and use CC = g++
I'm unsure whether there is another way since elinks was update to mozjs-52

Reply:
And I'm unsure on how to commit that into separated pull requests when there is spidermonkey.c needed to be updated several times and in that case the pull requests would get into conflicts...

Reply:
Sorry, please start from scratch.One pull request - one feature.